libs/components/packages/src/schematics/migrations/update-7/fix-mixed-case-imports.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { Tree } from '@angular-devkit/schematics';
+import { SchematicTestRunner } from '@angular-devkit/schematics/testing';
+
+describe('fix mixed case imports', () => {
+  let tree: Tree;
+  const runner = new SchematicTestRunner(
+    'schematics',
+    require.resolve('../migration-collection.json')
+  );
+  const angularJson = {
+    version: 1,
+    projects: {
+      test: {
+        projectType: 'application',
+        root: '',
+        architect: {},
+      },
+    },
+  };
+
+  function setupTest() {
```

Comment:
You can scope the variables needed for each test in the `setupTest` function. Then you won't need global variables that can easily get cumbersome. See: https://github.com/blackbaud/skyux/blob/main/libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.spec.ts#L79-L91

Reply:
Fixed the tests. Switched to a simpler find/replace because the ts printer was changing all the formatting.